apps/crc_interactive.py

```diff
@@ -77,6 +77,13 @@ def __init__(self) -> None:
         additional_args.add_argument('-f', '--feature', help='specify a feature, e.g. `ti` for GPUs')
         additional_args.add_argument('-o', '--openmp', action='store_true', help='run using OpenMP style submission')
 
+    def parse_time(time_str: str) -> time:
```

Comment:
Add function docstring with `Args`, `Return`, and `Raises` sections. Here is an automatically generated example:

```python
    """Parse a string representation of time in 'HH:MM:SS' format and return a time object

    Args:
        time_str: A string representing time in 'HH:MM:SS' format.

    Returns:
        time: A time object representing the parsed time.

    Raises:
        ArgumentTypeError: If the input string is not in the correct format or cannot be parsed.
    """
```